tests/hashing/subtests/avoid-cascade/webpack/webpack.config.js

```diff
@@ -33,10 +33,22 @@ module.exports = {
     moduleIds: 'hashed',
     // create a dedicated bootstrap + hash mapping bundle:
     runtimeChunk: 'single',
-    // extract all shared dependencies from entry bundles:
     splitChunks: {
+      // extract all shared dependencies from entry bundles:
       chunks: 'all',
       minSize: 0,
```

Comment:
@sokra is `enforce:true` for the runtime chunk enough that we could just drop `minSize` here?

Reply:
I initially removed it, but the acceptance criteria of the test is written in a way that requires `minSize: 0`:

> - Changing `utils.js` should only change the hash of its output file and the mapping.

Without minSize `utils.js` would be duplicated (because it's very small) in both entry chunks and will invalidate both.

But you could remove `minSize` if you make `utils.js` bigger, with 30k garbage...

Reply:
Yeah we originally had that (and I think still do for Parcel). Personally I prefer the `minSize:0` as long as its mentioned that's not a best-practise. All good - I just wasn't sure if it could be dropped after your change here.